moveit_planners/trajopt/src/problem_description.cpp

```diff
@@ -0,0 +1,567 @@
+
+#include <boost/algorithm/string.hpp>
+
+#include <ros/ros.h>
+
+#include <moveit/planning_interface/planning_interface.h>
+#include <moveit/planning_scene/planning_scene.h>
+#include <moveit_msgs/MotionPlanRequest.h>
+
+#include <trajopt/trajectory_costs.hpp>
+#include <trajopt_sco/expr_op_overloads.hpp>
+#include <trajopt_sco/expr_ops.hpp>
+#include <trajopt_utils/eigen_conversions.hpp>
+#include <trajopt_utils/eigen_slicing.hpp>
+#include <trajopt_utils/logging.hpp>
+#include <trajopt_utils/vector_ops.hpp>
+
+#include "trajopt_interface/problem_description.h"
+#include "trajopt_interface/kinematic_terms.h"
```

Comment:
Is this file copied in temporarily? If so, please explain in a TODO

Reply:
No, it is not temporary